tests/console/timezone.pm

```diff
@@ -24,20 +24,22 @@ use warnings;
 
 sub get_tz_data {
     my $save = shift;
+    my $filename = "testdata.zone";
     my $test_data = <<EOF;
 Rule    EU      1981    max     -       Mar     lastSun  1:00u  $save   S
 Rule    EU      1996    max     -       Oct     lastSun  1:00u  0       -
 Zone Europe/Rome -1:00 EU CE%sT
 EOF
-    return $test_data;
+    save_tmp_file($filename, $test_data);
+    return $filename;
 }
 
 sub set_data_and_validate {
-    my $tz_data = get_tz_data($_[0]);
-    script_run("echo '$tz_data' > $_[1]", 0);
-    assert_script_run("zic " . $_[1]);
+    my $file_tz_data = get_tz_data($_[0]);
```

Comment:
It'd be better to leave `get_tz_data()` as is and call `save_tmp_file() in `set_data_and_validate()`. I'd also leave the filename parameter there.

Reply:
done